min_rate: 10
linear_constant: 0.3
angular_constant: 2
